src/game/InstanceData.cpp

```diff
@@ -60,6 +86,39 @@ bool InstanceData::IsEncounterInProgress() const
     return false;
 }
 
+void InstanceData::OnPlayerEnter(Player *player)
+{
+    if (player->isGameMaster())
+    {
+        return;
+    }
+
+    QueryResultAutoPtr resultPlayerIds = QueryResultAutoPtr(NULL);
+    resultPlayerIds = RealmDataDatabase.PQuery("SELECT `player_ids` FROM `instance_times` WHERE `instance_id` = %i", instance->GetInstanceId());
+    if (resultPlayerIds)
+    {
+        Field* fields = resultPlayerIds->Fetch();
+        std::string playerIds = fields[0].GetString();
```

Comment:
Thanks for that. I think when I was writing this I was obsessed with the idea that the `player_ids` is theoretically an integer array and was trying to treat each player id individually.